csrc/utils.h

```diff
@@ -23,6 +23,7 @@
 #include <string>
 #include <type_traits>
 #include <typeinfo>
+#include <unordered_map>
```

Comment:
why would we need this header?

Reply:
I added this for a change that I later reverted. However, I keep it here for IWYU -- the current file already uses `std::unordered_map`.